br.com.barros.DevWidgets.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "br.com.barros.DevWidgets",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "devwidgets",
+    "separate-locales": false,
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--share=network"
+    ],
+    "modules": [
+      {
+        "name": "devwidgets",
+        "buildsystem": "simple",
+        "only-arches": [
+          "x86_64"
+        ],
+        "build-commands": [
+          "mkdir -p DevWidgets",
+          "tar -xf DevWidgets-Linux-Portable.tar.gz -C DevWidgets",
+          "cp -r DevWidgets /app/",
+          "chmod +x /app/DevWidgets/devwidgets",
+          "mkdir -p /app/bin",
+          "ln -s /app/DevWidgets/devwidgets /app/bin/devwidgets",
+          "mkdir -p /app/share/metainfo",
+          "cp -r br.com.barros.DevWidgets.metainfo.xml /app/share/metainfo/",
+          "mkdir -p /app/share/icons/hicolor/scalable/apps",
+          "cp -r icon.svg /app/share/icons/hicolor/scalable/apps/br.com.barros.DevWidgets.svg",
+          "mkdir -p /app/share/applications",
+          "cp -r br.com.barros.DevWidgets.desktop /app/share/applications/",
+          "mkdir -p /app/share/appdata",
+          "cp -r br.com.barros.DevWidgets.metainfo.xml /app/share/appdata/"
+        ],
+        "sources": [
+          {
+            "type": "file",
+            "path": "br.com.barros.DevWidgets.metainfo.xml"
+          },
+          {
+            "type": "file",
+            "path": "br.com.barros.DevWidgets.desktop"
+          },
+          {
+            "type": "file",
+            "path": "icon.svg"
+          },
+          {
+            "type": "file",
+            "path": "DevWidgets-Linux-Portable.tar.gz"
```

Comment:
definitely not. 
1. this file is not in the repo.
2. this file **SHOULD be downloaded from somewhere** and not be in the repo anyway.

Reply:
I'm really new to flatpak, how can I make it download from my github repo?